DataConnectors/NGINX/Connector_NGINX_agent.json

```diff
@@ -0,0 +1,149 @@
+{
+    "id": "NGINX HTTP Server",
```

Comment:
Id should be in this format NGINXHTTPServer (ProviderNameApplianceName).Please refer sample template https://raw.githubusercontent.com/Azure/Azure-Sentinel/master/DataConnectors/Templates/Connector_REST_API_template.json

Reply:
updated